src/utils/shell.ts

```diff
@@ -62,16 +59,13 @@ export const runCommand = (command, options = {}) => {
 
   // This ensures that an active spinner stays at the bottom of the commandline
   // even though the actual framework command might be outputting stuff
-  // @ts-expect-error TS(7006) FIXME: Parameter 'writeStream' implicitly has an 'any' ty... Remove this comment to see the full error message
-  const pipeDataWithSpinner = (writeStream, chunk) => {
-    if (spinner && spinner.isSpinning) {
+  const pipeDataWithSpinner = (writeStream: NodeJS.WriteStream, chunk: any) => {
```

Comment:
To be clear, chunk is literally typed as `any` in the NodeJS types. This isn't a lazy type 😄.